server/protocols/jmap-draft-integration-testing/jmap-draft-integration-testing-common/src/test/java/org/apache/james/jmap/JMAPAuthenticationTest.java

```diff
@@ -453,7 +453,7 @@ public void deleteMustReturnOKNoContentOnValidAuthorizationToken() throws Except
 
     @Category(BasicFeature.class)
     @Test
-    public void deleteMustInvalidAuthorizationOnCorrectAuthorization() throws Exception {
+    public void deleteMustInvalidAuthorizationOnCorrectAuthorization() {
         String continuationToken = fromGoodContinuationTokenRequest();
         zonedDateTimeProvider.setFixedDateTime(newDate);
```

Comment:
Please write a few more tests for what happens when we specify several authentication credentials?

Especially JWT + accessToken tests could be nice.